### DISCUSSION TITLE
Support for E-Book formats?

### DISCUSSION THREAD

**Discussion by ddeimeke** [jonaswinkler/paperless-ng#540] [Feature Requests]
Hi!

Would it be possible to add support for E-Book formats like Mobipocket or epub as well?

Being paperless could also mean considering reading books in a digital format.

For me it would be enough to support DRM-free books or books with a "soft"-DRM (digital watermark).

Cheers

Dirk

**Comment by Tooa** [comment]
Have you checked the ebook section of the selfhosted list [here](https://github.com/awesome-selfhosted/awesome-selfhosted#e-books-and-integrated-library-systems-ils)? Why do you want them in paperless?

**Reply by jonaswinkler** [reply]
I agree with that. There's better software out there for ebook management.

**Reply by ddeimeke** [reply]
I checked some of them.

Most of the applications are for clients. 

There is no good alternative for web based ebook management.

**Reply by Sblop** [reply]
Try ubooquity it works great. Lets keep paperless simple, as a document manager.

**Reply by ddeimeke** [reply]
I see, that we have different points of view.

For me an ebook is a document as well, which could be managed by a document management system.

It seems the other point of view is to see a DMS only as a system for documents like CVs, contracts, bills and such.

Question: What distinguishes a 30-page contract from a 30-page ebook?

**Reply by Sblop** [reply]
Hi Ddeimeke,
You have a fair point, and I like we keep this discussion clean and simple. :-)
I myself have thousands of ebooks and have found it difficult to manage. I love paperless to organize my docs. It works great, I can search. Currently I have 1400+ docs/statements/contracts etc and it fits in my world.
It is ofcause all subjective, however for my ebook collection i have been using calibre from linuxserver.io. Pair that with cablibre web and you have a strong although complicated but featurefull setup. I have migrated to ubooquity that handles my ebooks and my commics quite simple. I do not need to search in the text, as it has a quite nice folder structure. 
I do have a lot of PDF manuals in paperless, for all of our appliances/tvs etc, and I can search these and it works fine. - I do however still believe that ebook formats/formatting/reading is a subject for specified software like calibre or ubooquity.

**Reply by ddeimeke** [reply]
I get your point @Sblop 

Currently I do not think that I need fulltext search on ebooks as well. As mentioned "currently".

What I need is some sort of management tool for all things PDF, epub and mobi ("mobi" would be bonus, I only have a very little amount of Mobipocket/Kindle ebooks).

Thinking about the "currently" above, it would be great to have all "tables of content" searchable.

So, maybe paperless is the wrong tool for that. That's ok, I will keep on searching.
